aspnet/testing/integration-testing/sample/test/PrimeWeb.UnitTests/Services/PrimeService_IsPrimeShould.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+using PrimeWeb.Services;
+
+namespace PrimeWeb.UnitTests.Services
+{
+    public class PrimeService_IsPrimeShould
```

Comment:
Check https://github.com/aspnet/Home/wiki/Engineering-guidelines#unit-test-class-naming for our naming guidelines.